catroid/config/lint-baseline.xml

```diff
@@ -173,7 +173,7 @@
         errorLine2="         ~~~~~~">
         <location
             file="src\main\java\org\catrobat\catroid\ui\fragment\FormulaEditorFragment.java"
-            line="100"
+            line="101"
```

Comment:
please revert the changes to this file, see my next comment to know how to get rid of `EmptyFunctionBlock `lint warning